modules/async-icq/testing/simapp/upgrades/upgrades.go

```diff
@@ -55,7 +58,7 @@ func CreateV2UpgradeHandler(
 		// Migrate Tendermint consensus parameters from x/params module to a deprecated x/consensus module.
 		// The old params module is required to still be imported in your app.go in order to handle this migration.
 		baseAppLegacySS := paramskeeper.Subspace(baseapp.Paramspace).WithKeyTable(paramstypes.ConsensusParamsKeyTable())
-		baseapp.MigrateParams(ctx, baseAppLegacySS, &consensusparamskeeper)
+		baseapp.MigrateParams(sdk.UnwrapSDKContext(ctx), baseAppLegacySS, &consensusparamskeeper.ParamsStore)
```

Comment:
Async-ICQ v7.1.1 already migrated the baseAppLegacy params to the new consensus store (from deprecated x/params).

if we test the upgrade v7.1.1 -> v8.0.0 (local image), this is not needed. Only a empty `return mm.RunMigrations(ctx, cfg, vm)`
if we instead test v7.0.0->v8.0.0, this would still be required.

Reply:
doing v7.1.1 -> v8 since we already confirmed v7->v7.1.1